src/tox/tox_env/python/package.py

```diff
@@ -87,7 +87,11 @@ def default_wheel_tag(conf: Config, env_name: str | None) -> str:  # noqa: U100
             # python only code are often compatible at major level (unless universal wheel in which case both 2/3)
             # c-extension codes are trickier, but as of today both poetry/setuptools uses pypa/wheels logic
             # https://github.com/pypa/wheel/blob/master/src/wheel/bdist_wheel.py#L234-L280
-            run_py = cast(Python, run_env).base_python
+            try:
+                run_py = cast(Python, run_env).base_python
+            except NoInterpreter:
```

Comment:
**Note for reviewers:** Here. Also, you could previously get here by running e.g. `tox -e py33` on an environment that didn't provide `python3.3`.